src/schema/v2/artist/careerHighlights.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  GraphQLFieldConfig,
+  GraphQLBoolean,
+  GraphQLList,
+  GraphQLNonNull,
+  GraphQLString,
+} from "graphql"
+import { ResolverContext } from "types/graphql"
+import { CareerHighlightType } from "../careerHighlight/careerHighlight"
+
+const CareerHighlights: GraphQLFieldConfig<{ _id: string }, ResolverContext> = {
+  type: new GraphQLNonNull(
+    new GraphQLList(new GraphQLNonNull(CareerHighlightType))
+  ),
+  args: {
```

Comment:
any of these args might be required(I see partnerId might be GraphQLNonNull(GraphQLString))? And/or any room for some defaultValue too?